cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -870,15 +872,13 @@ func readCA(file string) ([]byte, error) {
 
 // createClientBuilders creates clientBuilder and rootClientBuilder from the given configuration
 func createClientBuilders(logger klog.Logger, c *config.CompletedConfig) (clientBuilder clientbuilder.ControllerClientBuilder, rootClientBuilder clientbuilder.ControllerClientBuilder) {
+	// Logger is not used, but the parameter is retained to avoid modifying the function signature.
+	_ = logger
```

Comment:
there's only a single local caller, so if we're not using the logger, go ahead and drop it from the signature